core/embed/rust/build.rs

```diff
@@ -277,6 +277,7 @@ fn generate_micropython_bindings() {
 
     // Write the bindings to a file in the OUR_DIR.
     bindings
+        .clang_arg(short_enums()) // make sure enums use the same size as in C
```

Comment:
Can we put this into `prepare_bindings`? Wouldn't have to repeat it & it already deals with some hw vs emulator issues.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/275a284b9c1d0bc28611df183c3df47c47f65374